CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 
 * Support for TLS-SNI-01 has been removed from all official Certbot plugins.
+* CLI flags `--tls-sni-01-port` and `--tls-sni-01-address` are now no-op, will
+  generate a deprecation warning if used, and will be removed soon.
+* Options `tls-sni` and `tls-sni-01` in `--preferred-challenges` flag are now no-op,
+  will generate a deprecation warning if used, and will be removed soon.
 
```

Comment:
Sorry I forgot about this. We should add an entry that the `--standalone-supported-challenges` flag has been removed.

Reply:
Forgot also. Done.